dotcom-rendering/src/components/MostViewedRight.tsx

```diff
@@ -29,47 +30,86 @@ interface Props {
 	stickToTop?: boolean;
 }
 
+interface MostViewedRightPayloadType {
+	tabs: FETrailTabType[];
+}
+
 export const MostViewedRight = ({
 	limitItems = 5,
 	stickToTop = false,
 }: Props) => {
+	const testVariant = useDeeplyReadTestVariant();
 	const endpointUrl =
-		'https://api.nextgen.guardianapps.co.uk/most-read-geo.json?dcr=true';
-	const { data, error } = useApi<FETrailTabType>(endpointUrl);
+		'https://api.nextgen.guardianapps.co.uk/most-read-with-deeply-read.json';
+	const { data, error } = useApi<MostViewedRightPayloadType>(endpointUrl);
 
 	if (error) {
-		window.guardian.modules.sentry.reportError(error, 'most-viewed-right');
+		window.guardian.modules.sentry.reportError(
+			error,
+			'most-viewed-deeply-read-right',
+		);
 		return null;
 	}
 
 	if (data) {
-		const trails: TrailType[] = data.trails
-			.slice(0, limitItems)
-			.map(decideTrail);
+		const mostReadTrails = data.tabs[0]?.trails
+			?.slice(0, limitItems)
+			?.map(decideTrail);
+		const deeplyReadTrails =
+			testVariant === 'deeply-read-and-most-viewed' ||
+			testVariant === 'deeply-read-only'
+				? data.tabs[1]?.trails?.slice(0, limitItems)?.map(decideTrail)
+				: undefined;
 
-		// Look I don't know why data-component is geo-most-popular either, but it is, ok? Ok.
 		return (
 			<div
 				css={[wrapperStyles, stickToTop && stickyStyles]}
 				data-component="geo-most-popular"
 			>
-				<StraightLines
-					cssOverrides={css`
-						display: block;
-					`}
-					count={4}
-					color={themePalette('--straight-lines')}
-				/>
-				<h3 css={headingStyles}>Most viewed</h3>
-				<ul data-link-name="Right hand most popular geo GB">
-					{trails.map((trail, index) => (
-						<MostViewedRightItem
-							key={trail.url}
-							trail={trail}
-							mostViewedItemIndex={index}
+				{testVariant !== 'deeply-read-only' && (
+					<>
+						<StraightLines
+							cssOverrides={css`
+								display: block;
+							`}
+							count={4}
+							color={themePalette('--straight-lines')}
+						/>
+						<h3 css={headingStyles}>Most viewed</h3>
+						<ul data-link-name="Right hand most popular geo GB">
+							{mostReadTrails?.map((trail, index) => (
+								<MostViewedRightItem
+									key={trail.url}
+									trail={trail}
+									mostViewedItemIndex={index}
+								/>
+							))}
+						</ul>
+					</>
+				)}
+				{(testVariant === 'deeply-read-only' ||
+					testVariant === 'deeply-read-and-most-viewed') && (
+					<>
+						<StraightLines
+							cssOverrides={css`
+								display: block;
+								margin-top: 30px;
```

Comment:
This will create an extra 30px of space in the `deeply-read-only` variant between the right ad slot and the Deeply Read component.

One idea might be to turn the Fragments <> into <section>'s and set a flex gap on the container i.e. `display: flex; flex-direction: column; gap: 30px;`. This way the space will only apply if there are multiple elements inside the div (on line 65).